src/scheduler/resv_info.cpp

```diff
@@ -412,10 +409,10 @@ query_reservations(int pbs_sd, server_info *sinfo, struct batch_status *resvs)
 							execvnode_ptr[degraded_idx - 1], sinfo, resresv_ocr->select);
 					else {
 						resresv_ocr->resv->orig_nspec_arr = {};
-						log_eventf(PBSEVENT_ERROR, PBS_EVENTCLASS_RESV,
-						           LOG_INFO, resresv->name,
-						           "%s: occurence %d has no execvnodes, using empty vector",
-						           __func__, j + 1);
+						log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_RESV,
+							   LOG_DEBUG, resresv->name,
+							   "%s: occurence %d has no execvnodes, using empty vector",
+							   __func__, j + 1);
```

Comment:
This is an error, so I was fine with it at the log level.  I just want the message to be more clear.  An admin won't understand what using an empty vector means.  Either just drop that part or explain what it means.